src/Uno.UI/UI/Xaml/Input/FocusManager.mux.cs

```diff
@@ -85,32 +85,32 @@ public partial class FocusManager : IFocusManager
 		/// "first" tab.  This is deemed better than getting stuck at the end.
 		/// (Reverse "first" and "last" for Shift-Tab.)
 		/// </summary>
-		private bool _canTabOutOfPlugin =
+		private const bool CanTabOutOfPlugin =
```

Comment:
Let's keep the original name, even if not changing, as it matches code from winui.

Reply:
Sure. I'll rename it back. I expect the current build to fail anyway due to a small [ca1841](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1841) violation which I'm fixing in https://github.com/unoplatform/uno/pull/9631.

So, I'll fix this after https://github.com/unoplatform/uno/pull/9631 is merged hoping for a green CI.